woocommerce/woocommerce
# Standardize config files

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/31133
Closes https://github.com/woocommerce/woocommerce/issues/31381

This PR basically just extracts package/plugin level configurations to the root level when it makes sense. Certain package/plugin level specific config items are not changed.

* `stylelint-config-wordpress` has been deprecated so it was replaced by `@wordpress/stylelint-config`
* `node-sass` has been replaced by `sass` as mentioned here https://github.com/woocommerce/woocommerce/issues/31381
* Bump node version to v16

### How to test the changes in this Pull Request:

* First install node v16. And also install pnpm (latest) globally.

1. Ensure all CI tests passes.
2. Locally try to run `pnpm install` and make sure no errors occur.
3. Try and build all the packages/plugin by running `pnpm nx run-many --target=build --all`. Ensure no errors.

### Other information:




### Changelog entry

### FOR PR REVIEWER ONLY:
